charts/honeycomb/templates/daemonset.yaml

```diff
@@ -62,6 +62,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.hostIP
+            {{- if gt (len .Values.additionalEnv) 0 -}}
+              {{- toYaml .Values.additionalEnv | nindent 12}}
+            {{- end }}
```

Comment:
Instead of a condition we can use `with`.  Something like

```yaml
{{- with .Values.additionalEnv }}
  {{- . | toYaml | nindent 12 }}
{{- end }}
```

Reply:
Thanks for the info. wasn't aware of this notion. I updated the code to use `with`